test/browser/components.test.js

```diff
@@ -1588,21 +1590,14 @@ describe('Components', () => {
 			}
 		}
 
+		// Statically create X element
 		const A = <X />;
 
 		class App extends Component {
 			constructor(props) {
 				super(props);
 				this.state = { i: 0 };
-			}
-
-			componentDidMount() {
```

Comment:
Syncrhonously calling rerender inside of `cDM` doesn't effectively replicate how Preact's diffing algorithm works (we batch updates and diff them later). Plus it causes an infinite loop so I rewrote this test.